Eat_and_Greet/src/comps/SplitButtons/style.tsx

```diff
@@ -0,0 +1,20 @@
+import { StyleSheet } from 'react-native';
+
+const SplitButtonStyles = StyleSheet.create({
+    button: {
```

Comment:
Nitpick, but should add a styles to this so that the text in each button (ex: 'User Information' is centered vertically within the button).